k8s/airflow/airflow-app.yaml

```diff
@@ -40,7 +40,7 @@ spec:
       initContainers:
         - name: airflow-xcom
           image: eu.gcr.io/airqo-250220/airqo-apache-airflow-xcom:latest
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
[only pull images when they are not present locally](https://github.com/airqo-platform/AirQo-api/pull/1330/commits/c2d47650b249e60a2f377a20e72b65ab6b66ee14)